app/build.gradle

```diff
@@ -23,13 +24,15 @@ dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
     androidTestCompile('com.android.support.test.espresso:espresso-core:2.2.2', {
         exclude group: 'com.android.support', module: 'support-annotations'
+        compile 'com.jakewharton:butterknife:8.7.0'
```

Comment:
Are you sure about nesting butteknife library into androidTestCompile for espresso?